cmd/buildkitd/main_oci_worker.go

```diff
@@ -262,7 +275,29 @@ func ociWorkerInitializer(c *cli.Context, common workerInitializerOpt) ([]worker
 	return []worker.Worker{w}, nil
 }
 
-func snapshotterFactory(commonRoot, name string) (runc.SnapshotterFactory, error) {
+func snapshotterFactory(commonRoot, name, address string) (runc.SnapshotterFactory, error) {
+	if address != "" {
+		logrus.Infof("proxy snapshotter address specified on %v", address)
+		newFunc := func(root string) (ctdsnapshot.Snapshotter, error) {
+			gopts := []grpc.DialOption{
+				grpc.WithInsecure(),
+				grpc.WithBackoffMaxDelay(3 * time.Second),
```

Comment:
WithBackoffMaxDelay is deprecated

Reply:
See https://github.com/containerd/containerd/pull/4053/files#diff-1b43e06d76b779c2d0cb9f11f9bf69f1L100

Reply:
Thanks for your feedback. I fixed it.